mediagalaxy.ro,"HTTPSConnectionPool(host='mediagalaxy.ro', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='mediagalaxy.ro', port=80): Read timed out. (read timeout=2)"
mediclinic.ae,"HTTPSConnectionPool(host='mediclinic.ae', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.mediclinic.ae', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
medicosdeelsalvador.com,"HTTPSConnectionPool(host='medicosdeelsalvador.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='medicosdeelsalvador.com', port=80): Read timed out. (read timeout=2)"
medinet.gov.vn,"HTTPSConnectionPool(host='medinet.gov.vn', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f386a6e62e0>, 'Connection to medinet.gov.vn timed out. (connect timeout=2)'))","HTTPConnectionPool(host='medinet.gov.vn', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f386b508d90>, 'Connection to medinet.gov.vn timed out. (connect timeout=2)'))"
medis.pt,"HTTPSConnectionPool(host='medis.pt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'medis.pt' doesn't match either of 'agentes.medis.com.pt', 'www.ocidental.pt', 'ocidental.pt', 'apps.ocidental.pt', 'agentes.ocidental.pt', 'clientes.medis.pt', 'www.medis.pt', 'corporate.medis.pt', 'empresas.medis.pt', 'beta.medis.pt', 'kiosk.medis.pt', 'prestadores.medis.com.pt', 'www.ocidentalpensoes.pt', 'beta.segurodirecto.pt', 'www.segurodirecto.pt', 'kiosk.segurodirecto.pt', 'beta.clinicamedis.pt', 'www.clinicamedis.pt'"")))","HTTPSConnectionPool(host='www.medis.pt', port=443): Read timed out. (read timeout=2)"
medvestnik.ru,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
medwave.cl,"HTTPSConnectionPool(host='medwave.cl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f38697be4f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='medwave.cl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f386a6e67f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mefos.hr,"HTTPSConnectionPool(host='mefos.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f386cbf3820>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mefos.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f386a78ddf0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
megashop.hr,"HTTPSConnectionPool(host='www.megashop.hr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.megashop.hr', port=443): Read timed out. (read timeout=2)"
megashop.si,"HTTPSConnectionPool(host='www.megashop.si', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.megashop.si', port=443): Read timed out. (read timeout=2)"
megaturg.ee,"HTTPSConnectionPool(host='www.megaturg.ee', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='megaturg.ee', port=443): Read timed out. (read timeout=2)"
meiji.ac.jp,"HTTPSConnectionPool(host='meiji.ac.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f386ca14100>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='meiji.ac.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3868b9d910>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
melody4arab.xyz,"HTTPSConnectionPool(host='melody4arab.xyz', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3869f904f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='melody4arab.xyz', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f386890f970>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
melonbooks.co.jp,"HTTPSConnectionPool(host='melonbooks.co.jp', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: EE certificate key too weak (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.melonbooks.co.jp', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
men.gov.ma,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
mendoza.gov.ar,"HTTPSConnectionPool(host='mendoza.gov.ar', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f386b022fd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mendoza.gov.ar', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3868bbf430>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
menlhk.go.id,"HTTPSConnectionPool(host='menlhk.go.id', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f386779ea60>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='menlhk.go.id', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3867af1f40>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
menofia.edu.eg,"HTTPSConnectionPool(host='menofia.edu.eg', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f386ca14df0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='menofia.edu.eg', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f386b4a0040>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
menoinfo.fi,"HTTPSConnectionPool(host='menoinfo.fi', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3867d2bbb0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='menoinfo.fi', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f386b0cfd30>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
menpan.go.id,"HTTPSConnectionPool(host='menpan.go.id', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='menpan.go.id', port=443): Read timed out. (read timeout=2)"
menupages.ie,"HTTPSConnectionPool(host='menupages.ie', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f386c22cfd0>, 'Connection to menupages.ie timed out. (connect timeout=2)'))","HTTPConnectionPool(host='menupages.ie', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f3868c05dc0>, 'Connection to menupages.ie timed out. (connect timeout=2)'))"
mep.go.cr,Exceeded 30 redirects.,Exceeded 30 redirects.
mercadopublico.cl,"HTTPSConnectionPool(host='mercadopublico.cl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f38675158b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mercadopublico.cl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f386a977f10>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mercedes.gob.ar,"HTTPSConnectionPool(host='mercedes.gob.ar', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f38697b3d30>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mercedes.gob.ar', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3866b54fd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
meta.vn,"HTTPSConnectionPool(host='meta.vn', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='meta.vn', port=80): Read timed out. (read timeout=2)"
meteo.it,"HTTPSConnectionPool(host='meteo.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))","HTTPSConnectionPool(host='meteo.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))"
meteo.lt,"HTTPSConnectionPool(host='meteo.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPConnectionPool(host='www.meteo.lt', port=80): Read timed out. (read timeout=2)"
meti.go.jp,"HTTPSConnectionPool(host='meti.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f38647883d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='meti.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f386bb9b430>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
metjekutindevut.nl,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
metrip.vn,"HTTPSConnectionPool(host='metrip.vn', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f386d797490>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='metrip.vn', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3864c85550>: Failed to establish a new connection: [Errno 111] Connection refused'))"
metro.tokyo.jp,"HTTPSConnectionPool(host='metro.tokyo.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f386bd729a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='metro.tokyo.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f386cb09700>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
metrouniv.ac.id,"HTTPSConnectionPool(host='metrouniv.ac.id', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='metrouniv.ac.id', port=443): Read timed out. (read timeout=2)"
metsalehti.fi,"HTTPSConnectionPool(host='metsalehti.fi', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f386a8838b0>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPSConnectionPool(host='www.metsalehti.fi', port=443): Read timed out. (read timeout=2)"
mext.go.jp,"HTTPSConnectionPool(host='mext.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3866fd5910>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mext.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f386d192130>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mfa.ee,"HTTPSConnectionPool(host='mfa.ee', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f386bfb1e50>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mfa.ee', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3867e19cd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mfa.gov.sg,"HTTPSConnectionPool(host='mfa.gov.sg', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'mfa.gov.sg' doesn't match 'www.mfa.gov.sg'"")))","HTTPSConnectionPool(host='mfa.gov.sg', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'mfa.gov.sg' doesn't match 'www.mfa.gov.sg'"")))"
mfa.gov.tr,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","HTTPSConnectionPool(host='mfa.gov.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
mhlw.go.jp,"HTTPSConnectionPool(host='mhlw.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f386ab19580>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mhlw.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3864dc8430>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mides.gub.uy,"HTTPSConnectionPool(host='mides.gub.uy', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f386bc8c700>, 'Connection to mides.gub.uy timed out. (connect timeout=2)'))","HTTPConnectionPool(host='mides.gub.uy', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f3869f0c670>, 'Connection to mides.gub.uy timed out. (connect timeout=2)'))"
mie.lg.jp,"HTTPSConnectionPool(host='mie.lg.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f386c987b80>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mie.lg.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3864958940>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
migdalor-news.co.il,"HTTPSConnectionPool(host='www.migdalor-news.co.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.migdalor-news.co.il', port=443): Read timed out. (read timeout=2)"
mikmak.co.il,"HTTPSConnectionPool(host='mikmak.co.il', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f386b8db670>, 'Connection to mikmak.co.il timed out. (connect timeout=2)'))","HTTPConnectionPool(host='mikmak.co.il', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f3867ec21c0>, 'Connection to mikmak.co.il timed out. (connect timeout=2)'))"
mikronis.hr,"HTTPSConnectionPool(host='mikronis.hr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.mikronis.hr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
milejemplos.com,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
milog.co.il,"HTTPSConnectionPool(host='milog.co.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='milog.co.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
min-mave.dk,"HTTPSConnectionPool(host='min-mave.dk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='min-mave.dk', port=443): Read timed out. (read timeout=2)"
min-saude.pt,"HTTPSConnectionPool(host='min-saude.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3867d7e400>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='min-saude.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f38640fe0a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
minci.gob.ve,"HTTPSConnectionPool(host='minci.gob.ve', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate (_ssl.c:1124)')))","HTTPConnectionPool(host='www.minci.gob.ve', port=80): Read timed out. (read timeout=2)"
minciencias.gov.co,"HTTPSConnectionPool(host='minciencias.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='minciencias.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
mindef.gob.bo,"HTTPSConnectionPool(host='mindef.gob.bo', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='mindef.gob.bo', port=80): Read timed out. (read timeout=2)"
mindop.sk,"HTTPSConnectionPool(host='mindop.sk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPConnectionPool(host='mindop.sk', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f386403b7f0>, 'Connection to mindop.sk timed out. (connect timeout=2)'))"
minec.gob.sv,"HTTPSConnectionPool(host='minec.gob.sv', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f3868fe2100>, 'Connection to minec.gob.sv timed out. (connect timeout=2)'))","HTTPConnectionPool(host='minec.gob.sv', port=80): Read timed out. (read timeout=2)"
minec.gob.ve,"HTTPSConnectionPool(host='minec.gob.ve', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f386d0e8c70>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='minec.gob.ve', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f386403b9a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mineco.gob.gt,"HTTPSConnectionPool(host='mineco.gob.gt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f386cfdf550>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mineco.gob.gt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3863d3e790>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mined.gob.sv,"HTTPSConnectionPool(host='mined.gob.sv', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f386bb9bac0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mined.gob.sv', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f386d3dcd60>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
minedu.gob.bo,"HTTPSConnectionPool(host='minedu.gob.bo', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))","HTTPSConnectionPool(host='minedu.gob.bo', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))"
minedu.sk,"HTTPSConnectionPool(host='minedu.sk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f386c493b50>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='minedu.sk', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f38649f7d00>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mineducacion.gov.co,"HTTPSConnectionPool(host='mineducacion.gov.co', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f386befd6d0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='mineducacion.gov.co', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3868fedc70>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
minetilbud.dk,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
minfin.bg,"HTTPSConnectionPool(host='www.minfin.bg', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.minfin.bg', port=443): Read timed out. (read timeout=2)"
mininterior.gov.co,"HTTPSConnectionPool(host='mininterior.gov.co', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f386a9d9f40>, 'Connection to mininterior.gov.co timed out. (connect timeout=2)'))","HTTPConnectionPool(host='mininterior.gov.co', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f3864958d00>, 'Connection to mininterior.gov.co timed out. (connect timeout=2)'))"
ministeriodegobierno.gob.ec,"HTTPSConnectionPool(host='ministeriodegobierno.gob.ec', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'ministeriodegobierno.gob.ec' doesn't match either of 'www.agricultura.gob.ec', 'agricultura.gob.ec'"")))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
ministeriodesalud.go.cr,"HTTPSConnectionPool(host='ministeriodesalud.go.cr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f386c0f5190>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ministeriodesalud.go.cr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3868d9d430>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
minsa.gob.pe,"HTTPSConnectionPool(host='minsa.gob.pe', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f386cba62e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='minsa.gob.pe', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f386a684490>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mintrabajo.gov.co,"HTTPSConnectionPool(host='mintrabajo.gov.co', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f38696f0640>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mintrabajo.gov.co', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3867ecff70>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mintransporte.gov.co,"HTTPSConnectionPool(host='mintransporte.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='mintransporte.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
minvu.cl,"HTTPSConnectionPool(host='minvu.cl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPConnectionPool(host='minvu.cl', port=80): Read timed out. (read timeout=2)"
minyu-net.com,"HTTPSConnectionPool(host='minyu-net.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))","HTTPSConnectionPool(host='minyu-net.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))"
miragenews.com,"HTTPSConnectionPool(host='www.miragenews.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.miragenews.com', port=443): Read timed out. (read timeout=2)"
mise.gov.it,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
missbloom.bg,"HTTPSConnectionPool(host='missbloom.bg', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f3867d9a760>, 'Connection to missbloom.bg timed out. (connect timeout=2)'))","HTTPConnectionPool(host='missbloom.bg', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f3863c9bc40>, 'Connection to missbloom.bg timed out. (connect timeout=2)'))"
missmundo.nl,"HTTPSConnectionPool(host='missmundo.nl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3864db0c40>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='missmundo.nl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f386cd36bb0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
misumi-ec.com,"HTTPSConnectionPool(host='misumi-ec.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f386c00f280>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='misumi-ec.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3863e42c40>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mizuhobank.co.jp,"HTTPSConnectionPool(host='mizuhobank.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3863ebaaf0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mizuhobank.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f38648075e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mj.pt,"HTTPSConnectionPool(host='mj.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f386cb89040>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mj.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3867dc7640>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mju.ac.th,"HTTPSConnectionPool(host='mju.ac.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3868bb3130>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mju.ac.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3866b5abe0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mku.edu.tr,"HTTPSConnectionPool(host='mku.edu.tr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f386aaa5d60>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mku.edu.tr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3863e26a00>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mladinska.com,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
mlit.go.jp,"HTTPSConnectionPool(host='mlit.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f386b5ca130>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mlit.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f386494e3d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mlsi.gov.cy,"HTTPSConnectionPool(host='mlsi.gov.cy', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f386c1d21c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mlsi.gov.cy', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3864886be0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mm.lv,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
moa.gov.cy,"HTTPSConnectionPool(host='moa.gov.cy', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='moa.gov.cy', port=443): Read timed out. (read timeout=2)"
mobilnisvet.com,"HTTPSConnectionPool(host='mobilnisvet.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='mobilnisvet.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
moda.sk,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
modacruz.com,"HTTPSConnectionPool(host='www.modacruz.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.modacruz.com', port=443): Read timed out. (read timeout=2)"
moe.edu.sg,"HTTPSConnectionPool(host='moe.edu.sg', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f38651a1f10>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='moe.edu.sg', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3869036400>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
moe.go.th,"HTTPSConnectionPool(host='moe.go.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3867afd940>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='moe.go.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f386c6863a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
moe.gov.jo,"HTTPSConnectionPool(host='moe.gov.jo', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='moe.gov.jo', port=443): Read timed out. (read timeout=2)"
moe.gov.lk,"HTTPSConnectionPool(host='moe.gov.lk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='moe.gov.lk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
moe.gov.my,"HTTPSConnectionPool(host='moe.gov.my', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: TLSV1_UNRECOGNIZED_NAME] tlsv1 unrecognized name (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.moe.gov.my', port=443): Read timed out. (read timeout=2)"
mof.gov.vn,"HTTPSConnectionPool(host='mof.gov.vn', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mof.gov.vn', port=443): Read timed out. (read timeout=2)"
mofa.go.jp,"HTTPSConnectionPool(host='mofa.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f38640fef70>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mofa.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f38644269d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
moh.gov.bh,"HTTPSConnectionPool(host='moh.gov.bh', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3867d7ea60>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='moh.gov.bh', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f386c68e9d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
moh.gov.jo,"HTTPSConnectionPool(host='moh.gov.jo', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='moh.gov.jo', port=80): Read timed out. (read timeout=2)"
moh.gov.my,"HTTPSConnectionPool(host='moh.gov.my', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'moh.gov.my' doesn't match '*.moh.gov.my'"")))","HTTPSConnectionPool(host='moh.gov.my', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'moh.gov.my' doesn't match '*.moh.gov.my'"")))"
moh.gov.vn,"HTTPSConnectionPool(host='moh.gov.vn', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='moh.gov.vn', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
mohre.gov.ae,"HTTPSConnectionPool(host='mohre.gov.ae', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.mohre.gov.ae', port=443): Read timed out. (read timeout=2)"
moi.gov.ae,"HTTPSConnectionPool(host='moi.gov.ae', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))","HTTPSConnectionPool(host='moi.gov.ae', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
